core/src/main/resources/hudson/model/UsageStatistics/global.properties

```
@@ -22,4 +22,8 @@
 
 statsBlurb=\
   Help make Jenkins better by sending anonymous usage statistics and crash reports to the Jenkins project
+statsBlurbFIPS=\
+  Help make Jenkins better by sending telemetry to the Jenkins project
+disabledBySystemProperty=\
+  The option to send anonymous usage statistics, crash reports and telemetry to the Jenkins project is disabled by a <a href="https://www.jenkins.io/doc/book/managing/system-properties/#hudson-model-usagestatistics-disabled">System Property</a>.
```

```suggestion
  The option to send anonymous usage statistics, crash reports and telemetry to the Jenkins project is disabled by a <a href="https://www.jenkins.io/redirect/usagestatistics-disabled">Java system property</a>.
```

We recommend redirect URLs from Jenkins, as they remain stable in the face of docs reorg (needs corresponding https://github.com/jenkins-infra/jenkins.io PR).